recipes/mpfr/all/CMakeLists.txt.in

```diff
@@ -1,15 +1,14 @@
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.19)
 project(cmake_wrapper)
 
-include("${{CMAKE_BINARY_DIR}}/conanbuildinfo.cmake")
-conan_basic_setup()
-
 # autotools project does not allow building a shared mpfr library (because of libtool limitation)
 
+find_package(gmp REQUIRED CONFIG)
+
 add_library(mpfr {mpfr_sources})
 target_compile_definitions(mpfr PRIVATE {definitions})
-target_include_directories(mpfr PRIVATE "${{CMAKE_BINARY_DIR}}" "${{CMAKE_BINARY_DIR}}/src" "${{CMAKE_CURRENT_SOURCE_DIR}}")
-target_link_libraries(mpfr PRIVATE ${{CONAN_LIBS}})
+target_include_directories(mpfr PRIVATE "${{CMAKE_BINARY_DIR}}" "${{CMAKE_BINARY_DIR}}/src" "${{CMAKE_CURRENT_SOURCE_DIR}}/src")
+target_link_libraries(mpfr PRIVATE gmp::gmp)
 if(BUILD_SHARED_LIBS)
     target_compile_definitions(mpfr PRIVATE MPFR_DLL)
 endif()
```

Comment:
Why keep this file? With `cmake_layout` these entrypoints are no longer needed no?

Reply:
Hi @Hopobcn,
Thanks for your review. I think they are still needed as I had to update the one path (adding "/src" at the end) to get things to build with Conan 2.0. So at this point my preference would be to leave it as is since I believe it is working. If this PR is accepted, we could do a follow-on optimization pass to see if any of this really can be eliminated. Thanks

Reply:
Ok works for me.